src/components/Link/Link.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+/* @jsx h */
+import {h} from 'hyperapp'
+import cn from 'classnames'
+import styles from './Link.css'
+
+import type {Props} from './flow-typed'
+import type {Children} from 'hyperapp'
+
+export const Link = ({className, href, target}: Props, children: Array<Children>) => (
+  <a
+    className={cn(styles.link, className)}
+    href={href}
+    target={target}>{children}</a>
+)
+
+export const LinkDiv = ({className}: {className?: string}, children: Array<Children>) => (
```

Comment:
I think we'll be use real link for `uploadcare--widget__file-name`, so we don't need that component